sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -514,6 +589,118 @@ static void _netif_list(kernel_pid_t iface)
     puts("");
 }
 
+#ifdef MODULE_LORA
+static int _netif_set_u32(kernel_pid_t iface, netopt_t opt, uint32_t context,
+                          char *u32_str)
+{
+    unsigned long int res;
+    bool hex = false;
+
+    if (_is_number(u32_str)) {
+        if ((res = strtoul(u32_str, NULL, 10)) == ULONG_MAX) {
+            puts("error: unable to parse value.\n"
+                 "Must be a 32-bit unsigned integer (dec or hex)\n");
+            return 1;
+        }
+    }
+    else {
+        if ((res = strtoul(u32_str, NULL, 32)) == ULONG_MAX) {
```

Comment:
I IIRC `strtoul()` isn't portable.

Reply:
Maybe but then the shell commands are already not portable, see [here](https://github.com/RIOT-OS/RIOT/blob/master/sys/shell/commands/sc_gnrc_netif.c#L524) (In fact I just copy pasted this function and replace uint16_t with uint32_t...)

Reply:
Alright.